### PR TITLE
Nuclear Missile Silo

### PR DESCRIPTION
**Pull request by Stev-the-third**
The Nuclear Missile Silo Bunker Thingamjig is a 2-z odyssey map. Should be nice and generic so storytellers can tell a variety of stories, including just blocking off the entire bunker and having a nice snowy town if they want to. 

StrongDMM-2024-12-13 09 59 43
image

And the sub-maps:
image

Future plans for a later PR include a day/night cycle and maybe weather

### CODE REVIEW

**Comment by Stev-the-third** [comment]
!wip

**Comment by Stev-the-third** [comment]
!review

File: code/game/objects/structures/light_pole.dm [hunk 1]
```diff
@@ -13,6 +13,21 @@
...
+	name = "light pole"
+	desc = "A tall light source. The bulb appears to be decayed."
+	icon = 'icons/effects/32x96.dmi'
+	icon_state = "rustlamp_l"
+	anchored = TRUE
+	density = TRUE
+	light_wedge = LIGHT_OMNI
+	light_color = LIGHT_COLOR_DECAYED
+	light_range = 8
+	light_power = 8
```
Comment (DreamySkrell) [inline code/game/objects/structures/light_pole.dm:26]: do not re-define all these vars, they're already defined in the parent type
only re-define what is changed from the parent type, in this case desc and light_color

File: code/modules/paperwork/stamps.dm [hunk 2]
```diff
@@ -50,6 +50,10 @@
 /obj/item/stamp/denied
 	name = "\improper DENIED rubber stamp"
 	icon_state = "stamp-deny"
+//----- Finally. I can do papers please LARP.
```
Comment (DreamySkrell) [inline code/modules/paperwork/stamps.dm:53]: remove this comment

File: code/game/objects/structures/light_pole.dm (hunk 1 above)
Comment (Stev-the-third) [reply code/game/objects/structures/light_pole.dm]: I think this is right now.

**Comment by hazelrat** [comment]
The new files used for the map don't seem to have been committed into aurorastation.dme?

**Comment by Stev-the-third** [comment]
> The new files used for the map don't seem to have been committed into aurorastation.dme?

aough.

**Comment by hazelrat** [comment]
image
If the turfs here are meant to be dark, I'd recommend laying light sources of some kind along the path from the Intrepid-sized shuttle landing zones. I think people might not easily register which way they're meant to go. Those light poles could work, maybe a dimmer variant?

image

The Intrepid is south-facing, so when it docks here it'll have its airlock be two turfs away from an impassable wall. Not good for foot traffic, it's a bit like opening a door to a brick wall. I think it'd be preferable to move these LZs a little south so a trodden area can be set up outside of the airlock where people can set up an FOB, with a path linking down to the town.

**Comment by hazelrat** [comment]
image
The overmap project should probably be named according to the planet it's on, not by the town in the valley.

**Comment by Stev-the-third** [comment]
!wip

**Comment by DreamySkrell** [comment]
when tests are passing, ping me on discord and then I'll review this

File: code/__DEFINES/access.dm [hunk 3]
```diff
@@ -824,6 +824,8 @@
 	id = ACCESS_LONE_SPACER
 	desc = "Independent Skiff Access"
 
+// for Odyssey scenarios
```
Comment (DreamySkrell) [inline code/__DEFINES/access.dm:827]: remove this comment, accesses are taken on a first available basis, we don't reserve numbers for things

File: maps/away/scenarios/nuclear_silo/missile_silo.dm [hunk 4]
```diff
@@ -0,0 +1,22 @@
...
+	name = "missile silo blast door"
+	desc = "An enormous reinforced plasteel silo door. It is mechanically operated and can not be forced non-destructively."
+	icon = 'icons/effects/missile_silo.dmi'
+	icon_state = "silo"
+	density = 1
+	anchored = 1
+
+/obj/structure/icbm
+	name = "RS-20 nuclear ballistic missile"
+	desc = "A Zavodskoi-designed hypersonic missile carrying a nuclear payload. With a maximum speed of over 20% the speed of light in vacuum or Mach 25 in atmosphere, interplanetary travel capability, and an accuracy to the meter, this missile forms a cornerstone of arsenals galaxy-wide, able to function as both an anti-orbital weapon and an intercontinental nuclear bomb, delivered anywhere on a planet or in a solar system. The nuclear warhead on this has enough power to level a small moon."
```
Comment (DreamySkrell) [inline maps/away/scenarios/nuclear_silo/missile_silo.dm:11]: "The nuclear warhead on this has enough power to level a small moon" sounds silly for just one missile

even just small moons are massive

**Comment by DreamySkrell** [comment]
image
this should not be 255x255, it should only be as big as the submaps in it (+ a few tiles on the edges for readability)

**Comment by DreamySkrell** [comment]
image
horizon armory access on these lockers

**Comment by DreamySkrell** [comment]
image
these submaps should be set up like horizon's small warehouse submaps, where only the warehouse contents are overlayed on top of the horizon map, to avoid repetition of the turfs and pipes and etc in every submap variation

**Comment by DreamySkrell** [comment]
image
fake objects are not allowed

**Comment by DreamySkrell** [comment]
image
define a new dummy terminal type instead of using the base terminal type with these vars mapedited

**Comment by DreamySkrell** [comment]
image
use the /off lamp subtype instead

**Comment by DreamySkrell** [comment]
image

File: code/__DEFINES/access.dm (hunk 3 above)
Comment (Stev-the-third) [reply code/__DEFINES/access.dm:827]: I will remove this comment, but my thought process behind it was that there's a comment of "Misc" after the horizon areas end, and the third party ships start.

**Comment by Stev-the-third** [comment]
Changes should be made.

**Review by DreamySkrell — APPROVED** [review]

**Comment by DreamySkrell** [comment]
good map

**Comment by DreamySkrell** [comment]
actually. please add some screenshots of the map to the PR description

**Pull request merged**